docs/models/dti.rst

```diff
@@ -4,7 +4,171 @@
 Diffusion tensor imaging (DTI)
 ##############################
 
+DTI Foundational Papers
+=======================
 
+**DTI Concept Introduction**: Basser et al. introduced DTI as a new MRI 
+modality that computes a Gaussian diffusion tensor per voxel, yielding eigenvalues 
+and eigenvectors that describe 3D water diffusion :cite:p:`basser1994`. 
+This enabled mapping of fiber orientations in tissue and orientation-independent 
+scalar measures like trace (sum of eigenvalues, related to Mean Diffusivity, MD). 
+It established the idea that tensors encode more microstructural information 
+than single-direction diffusivities.
+
+**First Anisotropy Indices**: Pierpaoli and Basser defined fractional anisotropy 
+(FA), a normalized 0–1 index quantifying how anisotropic (directionally dependent)
+diffusion is :cite:p:`pierpaolibasser1996`. 
+They demonstrated that earlier methods (diffusion measured in just a few directions) 
+underestimated anisotropy, and introduced rotationally invariant metrics (FA, relative 
+anisotropy, etc.) to robustly characterize white matter integrity. This work also 
+cautioned that noise can bias anisotropy measures requiring eigenvalue ordering.
+
+**First Human DTI Maps**: Pierpaoli et al. acquired the first DTI scans of the human 
+brain, mapping principal diffusion directions and magnitudes 
+:cite:p:`pierpaoli1996`. They observed that water diffuses ~3 times faster along 
+axonal fibers than perpendicular to them in highly coherent tracts (e.g. corpus 
+callosum) than in regions with crossing or less organized fibers.
```

Comment:
```suggestion
callosum).
```
This sentence gets a bit long and convoluted. The 3 times faster is for one comparison (along fibers relative to perpendicular), so it's not clear what the word "than" means in this last fragment. I would suggest to remove the last fragment and add another sentence to discuss differences between regions, if needed.